libcontainer/cgroups/fs/stats_util_test.go

```diff
@@ -131,3 +131,41 @@ func expectPageUsageByNUMAEquals(t *testing.T, expected, actual cgroups.PageUsag
 		t.Errorf("Expected hierarchical total page usage by NUMA %#v but found %#v", expected.Hierarchical.Unevictable, actual.Hierarchical.Unevictable)
 	}
 }
+
+func expectCPUUsageEqual(t *testing.T, expected, actual cgroups.CpuUsage) {
+	if expected.TotalUsage != actual.TotalUsage {
```

Comment:
_Maybe_ you could just use [reflect.DeepEqual](https://golang.org/pkg/reflect/#DeepEqual) here (or directly in the `TestCpuacctStats`). If there are other places like this, they can be converted to use DeepEqual, too (as a separate preceding patch).